tests/Type/GeneralTypeTest.php

```diff
@@ -63,13 +72,20 @@ public static function dataFileAsserts(): iterable
         yield from self::gatherAssertTypes(__DIR__ . '/data/view-exists.php');
         yield from self::gatherAssertTypes(__DIR__ . '/data/view.php');
         yield from self::gatherAssertTypes(__DIR__ . '/data/where-relation.php');
-        yield from self::gatherAssertTypes(__DIR__ . '/data/bug-1997.php');
-        yield from self::gatherAssertTypes(__DIR__ . '/data/bug-1819.php');
 
         if (version_compare(LARAVEL_VERSION, '11.28.0', '>=')) {
             yield from self::gatherAssertTypes(__DIR__ . '/data/model-collections-l11-28.php');
         }
 
+        // 11.100 is an artificial constraint until 12.0.0 is released
+        if (version_compare(LARAVEL_VERSION, '11.0.0', '>=') && version_compare(LARAVEL_VERSION, '11.100.0', '<')) {
```

Comment:
```suggestion
        if (laravel_version_compare('11.0.0', '>=') && laravel_version_compare('12.0.0', '<')) {
```

Don't forget to import `use function Orchestra\Testbench\laravel_version_compare;`

Reply:
@crynobone, likely need to add 12.x here as well:


![image](https://github.com/user-attachments/assets/d9d5c5e9-e642-4710-9669-2edf1372dd9e)